test/test_dynproc.py

```diff
@@ -164,14 +164,27 @@ def testJoin(self):
         size = MPI.COMM_WORLD.Get_size()
         rank = MPI.COMM_WORLD.Get_rank()
         server = client = address = None
-        # crate server/client sockets
+        host = socket.gethostname()
+        addresses = socket.getaddrinfo(host, None, type=socket.SOCK_STREAM)
```

Comment:
On Python 2, `getaddrinfo()` does not accept keyword arguments.

```suggestion
        addresses = socket.getaddrinfo(host, None, 0, socket.SOCK_STREAM)
```